libraries/standard/mqtt/CMakeLists.txt

```diff
@@ -46,41 +37,30 @@ if( ${IOT_BUILD_TESTS} )
 
     # This test library is used to mock MQTT connections.
     add_library( iot_mqtt_mock
-                 test/mock/iot_tests_mqtt_mock.c
+                 ${MQTT_TEST_MOCK_SOURCES}
                  test/mock/iot_tests_mqtt_mock.h
```

Comment:
Why do we use a variable for the .c but not the .h on line 41?

Reply:
It is because the header files can be included in the repository by adding them in the include paths. It is already covered by the variable MQTT_TEST_MOCK_INCLUDE_PUBLIC_DIRS. That is the reason why I have not included .h file in the MQTT_TEST_MOCK_SOURCES variable.